src/core/utils.ts

```diff
@@ -187,6 +191,23 @@ export function hasVisibleLegendItems(options: Highcharts.Options) {
   });
 }
 
+export function hasVisibleSecondaryLegendItems(options: Highcharts.Options) {
+  const yAxes = Array.isArray(options.yAxis) ? options.yAxis : options.yAxis ? [options.yAxis] : [];
```

Comment:
I'd add more code comments to explain why do we target yAxis only, and that we only support two axes at most.